docs-src/modules/developing/pages/specFiles.adoc

```diff
@@ -0,0 +1,241 @@
+:tabs-sync-option:
+
+= Spec Files 
+
+In addition to modifiying resources directly, you can also use a *spec file* to describe the desired state of Kaskada.
```

Comment:
Do we intend to include the actual spec of each resource? For example, the [argument references](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#argument-reference) for AWS S3 Buckets describe each of the fields/requirements.

Reply:
Eventually maybe yes, but here I just cut & pasted the text and format over from the old specFile sections in the `table` and `view` docs and made the minimum changes. I'd prefer to make larger SpecFile doc changes in a separate PR.